SECURITY.md

```diff
@@ -3,6 +3,76 @@
 Due to the nature of what we are doing, fwupd takes security very seriously.
 If you have any concerns please let us know.
 
+## Considerations For Secure Boot
+
+In most secure boot configurations `fwup*.efi` will be signed and run by a first-stage bootloade
+(e.g. Shim) and it is thus critically important that fwup-efi is not allowed to run unauthenticated
+or untrusted code.
+
+The following sections contain considerations about what the EFI binary actually does, which
+hopefully should give enough assurance that using `fwupd.efi` is thus no threat to the chain of
+trust required by secure boot.
+
+### General Logic Flow
+
+* Find and validate update state variables:
+
+* Read the list of EFI variables, and match the ones with the fwupd GUID.
```

Comment:
It's probably worth either linking to the fwupd GUID in code or putting it in parenthesis

Reply:
Good idea, fixed.